### PR TITLE
Consistently use `Phase` instead of `Step`

### PR DESCRIPTION
**Pull request by masih**
Use the term `Phase` consistently across the repo to refer to progress phases of GPBFT.

FIP update to follow.

### CODE REVIEW

File: cmd/f3/msgdump/envelope.go
Comment (masih) [inline cmd/f3/msgdump/envelope.go:42]: Note this is a breaking change if we ever want to analyse old parquet logs.

**Review by Stebalien — APPROVED** [review]

**Comment by Stebalien** [comment]
LGTM, but I'm going to wait for @Kubuxu to sign off before merging.

File: cmd/f3/msgdump/envelope.go
Comment (Kubuxu) [reply cmd/f3/msgdump/envelope.go:42]: The analysis is done in duckdb, so the queries will have to be updated, but it is fine.

**Pull request merged**
